cpp/inc/bond/core/apply.h

```diff
@@ -16,26 +16,26 @@ namespace detail
 {
 
 /// @brief Apply transform to serialized struct wrapped in bonded<T>
-template <typename Transform, typename T, typename Reader>
+template <typename Protocols, typename Transform, typename T, typename Reader>
```

Comment:
That is intentional, so that the user can explicitly pass a custom protocol list while arguments are still deduced implicitly (e.g. `bond::Apply<MyProtocols>(transform, value)`).